File: src/main/java/dbconnection/model/ContactTest.java
package dbconnection.model;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;

/**
 * Database table
 */
@Entity
public class ContactTest {
    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    private Long objid;

    private String firstName;

    private String lastName;

    public Long getObjid() {
        return objid;
    }

    public void setObjid(Long objid) {
        this.objid = objid;
    }

    public String getFirstName() {
        return firstName;
    }

    public void setFirstName(String firstName) {
        this.firstName = firstName;
    }

    public String getLastName() {
        return lastName;
    }

    public void setLastName(String lastName) {
        this.lastName = lastName;
    }
}

File: src/main/java/connectionKotlin/db/model/ContactTest.kt
package connectionKotlin.db.model

import javax.persistence.Column
import javax.persistence.Entity
import javax.persistence.GeneratedValue
import javax.persistence.GenerationType.AUTO
import javax.persistence.Id

@Entity
class ContactTest {
    @Id @GeneratedValue(strategy = AUTO)
    var objid: Long? = null

    @Column
    var firstName: String? = null

    @Column
    var lastName: String? = null
}